src/Generators/MigrationGenerator.php

```diff
@@ -156,13 +156,26 @@ protected function buildDefinition(Model $model)
         return trim($definition);
     }
 
-    protected function buildPivotTableDefinition(array $segments, $dataType = 'unsignedBigInteger')
+    protected function buildPivotTableDefinition(array $segments, $dataType = 'foreignId')
```

Comment:
Is the `dataType` parameter ever sent as something? Just wondering why it's even needed or if it could be set locally. I suppose maybe it could be a `uuid`?

Reply:
I'll leave this one up to you!

Reply:
Well, you're changing the default, so I'd like to know why/how it's used.

Reply:
You're right, I didn't quite catch your question the first time. The default value should be reverted and the value of the datatype when `use_constraints` is enabled the type should be `foreignId`.